themes/codeforpoland/public/js/lib/bootstrap-tagsinput.js

```diff
@@ -79,7 +79,8 @@
         return;
 
       // Trim value
-      if (typeof item === "string" && self.options.trimValue) {
+      if (typeof item === "string") {
+        console.log('trimming', item)
```

Comment:
Not sure if you meant to leave this in, but it jumped out to me because certain versions of IE don't have a `console` (this _appears_ to be a clientside JS file).